upup/pkg/fi/cloudup/awstasks/iaminstanceprofile.go

```diff
@@ -93,17 +95,21 @@ func (e *IAMInstanceProfile) Run(c *fi.Context) error {
 	return fi.DefaultDeltaRunMethod(e, c)
 }
 
-func (s *IAMInstanceProfile) CheckChanges(a, e, changes *IAMInstanceProfile) error {
+func (_ *IAMInstanceProfile) CheckChanges(a, e, changes *IAMInstanceProfile) error {
```

Comment:
Why did we change the receiver?

Reply:
It may be incorrect, but all of the other IAM tasks have `func (s *IAMInstanceProfile)`

Reply:
It might've been my IDE complaining. I just reverted this change